src/main/java/com/google/sps/servlets/AuthFilter.java

```diff
@@ -13,68 +13,65 @@
 // limitations under the License.
 
 package com.google.sps.servlets;
-
-import javax.servlet.FilterChain;
-import javax.servlet.Filter;
-import javax.servlet.FilterConfig;
-import javax.servlet.annotation.WebFilter;
-import javax.servlet.annotation.WebInitParam;
-import javax.servlet.ServletRequest;
-import javax.servlet.ServletResponse;
-import javax.servlet.ServletException;
+import com.google.auth.oauth2.GoogleCredentials;
 import com.google.firebase.FirebaseApp;
 import com.google.firebase.auth.FirebaseAuth;
 import com.google.firebase.FirebaseOptions;
 import com.google.firebase.auth.FirebaseToken;
+import com.google.firebase.auth.FirebaseAuthException;
 import com.google.gson.Gson;
+import com.google.sps.servlets.MutableHttpServletRequest;
+import com.google.sps.servlets.FirebaseAppInit;
 import java.io.IOException;
-import java.io.FileInputStream;
+import java.util.List;
+import javax.servlet.annotation.WebFilter;
+import javax.servlet.annotation.WebInitParam;
 import javax.servlet.annotation.WebServlet;
+import javax.servlet.FilterChain;
+import javax.servlet.Filter;
+import javax.servlet.FilterConfig;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.ServletException;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import org.apache.commons.io.IOUtils;
-import java.io.*;
-import java.util.List;
-import com.google.auth.oauth2.GoogleCredentials;
-import com.google.firebase.auth.FirebaseAuthException;
-import com.google.sps.servlets.MutableHttpServletRequest;
-import com.google.sps.servlets.FirebaseAppInit;
-
 
 @WebFilter(urlPatterns= {"/chatrooms", "/user", "/users", "/getChatroom", "/messages"})
 public class AuthFilter implements Filter {
-
-    FirebaseAppInit firebaseAppInit;
-    FirebaseApp firebaseApp;
+    private FirebaseAppInit firebaseAppInit;
+    private FirebaseApp firebaseApp;
 
     // Create an instance of FirebaseApp.
     public AuthFilter() throws IOException {
         firebaseAppInit = new FirebaseAppInit();
         firebaseApp = firebaseAppInit.initializeFirebaseApp();
     }
 
-    public void init(FilterConfig config) {}
+    public void init(FilterConfig config) throws ServletException {}
```

Comment:
This function doesn't do anything, does it need to throw exception?

Reply:
no. i will correct it